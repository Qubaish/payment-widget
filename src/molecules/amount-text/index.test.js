import React from 'react';
import { render } from '@testing-library/react';
import AmountText from './index';

describe('<AmoutText />', () => {
    it('checking snapshot', () => {
        const { asFragment } = render(<AmountText amount="1000"/>);
        expect(asFragment()).toMatchSnapshot();
    });
    it('should render a amount text component', () => {
        const { getByTestId } = render(<AmountText amount="1000"/>);
        const elem = getByTestId('amount');
        expect(elem.children.length).toBe(2);
    });
});