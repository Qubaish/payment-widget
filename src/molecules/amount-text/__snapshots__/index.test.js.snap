// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AmoutText /> checking snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxhCb laRzfC"
    data-testid="amount"
  >
    <p
      class="sc-AxirZ fqAQvC"
      data-testid="text"
    >
      1000
    </p>
    <button
      aria-label="close button"
      class="sc-AxiKw dCgSdz"
      data-testid="btn"
      type="sm"
    >
      close
    </button>
  </div>
</DocumentFragment>
`;
